### PR TITLE
Apple.FaceTimeCameraUpdate version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111867&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Manual Validation failed with:
image

File: manifests/a/Apple/FaceTimeCameraUpdate/1.0/Apple.FaceTimeCameraUpdate.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Apple.FaceTimeCamerUpdate
```
Comment (stephengillie) [inline manifests/a/Apple/FaceTimeCameraUpdate/1.0/Apple.FaceTimeCameraUpdate.installer.yaml:4]:
```suggestion
PackageIdentifier: Apple.FaceTimeCameraUpdate
```

File: manifests/a/Apple/FaceTimeCameraUpdate/1.0/Apple.FaceTimeCameraUpdate.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.FaceTimeCamerUpdate
```
Comment (stephengillie) [inline manifests/a/Apple/FaceTimeCameraUpdate/1.0/Apple.FaceTimeCameraUpdate.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Apple.FaceTimeCameraUpdate
```

File: manifests/a/Apple/FaceTimeCameraUpdate/1.0/Apple.FaceTimeCameraUpdate.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Apple.FaceTimeCamerUpdate
```
Comment (stephengillie) [inline manifests/a/Apple/FaceTimeCameraUpdate/1.0/Apple.FaceTimeCameraUpdate.yaml:4]:
```suggestion
PackageIdentifier: Apple.FaceTimeCameraUpdate
```

**Comment by Trenly** [comment]
> Manual Validation failed with: image

[Policy] Hardware

**Pull request closed without merging**